machine/install-machine.md

```diff
@@ -66,11 +70,11 @@ To install the scripts, copy or link them into your `/etc/bash_completion.d` or
 `/usr/local/etc/bash_completion.d` directory. To enable the `docker-machine` shell
 prompt, add `$(__docker_machine_ps1)` to your `PS1` setting in `~/.bashrc`.
 
-    PS1='[\u@\h \W$(__docker_machine_ps1)]\$ '
+```
+PS1='[\u@\h \W$(__docker_machine_ps1)]\$ '
+```
 
-You can find additional documentation in the comments at the <a
-href="https://github.com/docker/machine/tree/master/contrib/completion/bash"
-target="_blank">top of each script</a>.
+You can find additional documentation in the comments at the [top of each script](https://github.com/docker/machine/tree/master/contrib/completion/bash){: target="_blank" class="_"}.
 
 ### How to uninstall
```

Comment:
I changed this to "How to uninstall Docker Machine" ... I think it's nicer not to dictate an uninstall :-)  and it is in keeping with the Toolbox topics naming (e.g., https://docs.docker.com/toolbox/toolbox_install_mac/#how-to-uninstall-toolbox)